### DISCUSSION TITLE
why doesn't rednotebook use the standardized Markdown language?

### DISCUSSION THREAD

**Discussion by nutpantz** [jendrikseipp/rednotebook#711] [Ideas]
https://www.markdownguide.org/

why doesn't rednotebook use the standardized Markdown language over its own?  
  
it might bring more users, and when they learn to use it has uses with a lot of other apps.

**Comment by curioussushiroll** [comment]
As far as I know, markdown is being considered right now.
[Markdown issue](https://github.com/jendrikseipp/rednotebook/issues/560)
[TODO](https://github.com/jendrikseipp/rednotebook/blob/master/TODO.md#deferred-features)
